sdk/appconfiguration/app-configuration/test/public/index.spec.ts

```diff
@@ -30,8 +37,8 @@ describe("AppConfigurationClient", () => {
 
   describe("simple usages", () => {
     it("Add and query a setting without a label", async () => {
-      const key = recorder.getUniqueName("noLabelTests");
-
+      recorder.variable("noLabelTests", `noLabelTests-${getRandomNumber()}`);
+      key = recorder.variable("noLabelTests");
```

Comment:
This binding of key is a little weird to me. The name is bound in the outer describe scope, assigned in the beforeEach, and then reassigned here?

If this one test is unique, I would avoid re-using the name from the outer scope and just create a local constant. I'm not sure I follow what this test is doing with `key`, though.